node/bft/src/bft.rs

```diff
@@ -322,6 +322,13 @@ impl<N: Network> BFT<N> {
         committee: Committee<N>,
         current_round: u64,
     ) -> bool {
+        // Retrieve the authors for the current round.
```

Comment:
Should the function comment `Returns 'true' under one of the following conditions:` also be corrected, or was it already correct?

Reply:
I've updated the documentation to be more clear.